### DISCUSSION TITLE
Differential expression after harmony

### DISCUSSION THREAD

**Discussion by akshitupneja** [stuart-lab/signac#640] [Q&A] [answered]
Hello,
Thanks for this tool!

We have integrated snATAC batches with harmony using the peaks assay, and called FindNeighbors and FindClusters in harmony space. Do you recommend running FindAllMarkers for cluster annoation (using gene activity) and performing other DE analyses using the integrated object?
In the Seurat tutorial, you suggest doing these analyses on the unintegrated objects. Any advice would be appreciated.

**Comment by timoast** [comment, marked as answer]
> Do you recommend running FindAllMarkers for cluster annoation (using gene activity) and performing other DE analyses using the integrated object?

For Harmony, the correction is performed on the LSI components. We also correct the LSI components in our recommended scATAC integration vignette: https://satijalab.org/signac/articles/integrate_atac.html. Since you're only correcting the LSI components, not the peak matrix itself, there's no concept of using an integrated peak matrix for differential testing (which we don't recommend anyway). The workflow would be to cluster based on the integrated LSI components, and then continue using the original peak matrix for downstream analysis tasks like differential accessibility. Whether you want to annotate clusters based on differential gene activity, or some other way, is completely up to you.
